test/README.md

```diff
@@ -32,9 +33,11 @@ cargo xtask integration-test
 ### Virtualized
```

Comment:
While we're here, can we make Virtualized the default way of running the tests - via xtask of course.
This has long been on the TODO list since we know the tests don't always clean up after themselves.